app/controllers/api/v0/events_controller.rb

```diff
@@ -7,20 +7,11 @@ def create
 
     inbound_binding.events.each do |event|
       event.data.user_uuid = CompactUuid.pack(current_user_uuid) if current_user_uuid
-
-      avro_schema_type = avroturf_path(data: event.data)
-      avro_encoded_data = KafkaAvroTurf.instance.encode(event.data.to_hash, schema_name: avro_schema_type)
+      avro_encoded_data = KafkaAvroTurf.instance.encode(event.data.to_hash, schema_name: event.data.type)
```

Comment:
would we ever need to pass in an explicit version here?

Reply:
so..the "subject" names in SR will be our semantic name_version right?

Reply:
Yes